Onboarding.md

```diff
@@ -0,0 +1,228 @@
+
+`builtin-ledger-grpc-client-lib`
+Builtin Ledger GRPC Client Library.
+[README](./packages/builtin-ledger-grpc-client-lib/README.md)
+
+`builtin-ledger-grpc-svc`
+Builtin Ledger GRPC Service.
+[README](./packages/builtin-ledger-grpc-svc/README.md)
+
+`grpc-client-lib`
+GRPC Client Library.
+[README](./packages/grpc-client-lib/README.md)
+ 
+`grpc-svc`
+GRPCS Service.
+[README](./packages/grpc-svc/README.md)
+ 
+`public-types-lib`
+Accounts and Balances BC Public Types Library.
+[README](packages/public-types-lib/README.md) 
+
+`shared-mocks-lib`
+Mock implementation used for testing.
+[README](./packages/shared-mocks-lib/README.md)
+
+## 3. <a name='InstallingandBuilding'></a>Installing and Building
+
+Firstly, clone your fork of the `accounts-and-balances-bc` onto your local machine:
+```bash
+git clone https://github.com/<your_username>/accounts-and-balances-bc.git
+```
+
+Then `cd` into the directory and install the node modules:
+```bash
+cd accounts-and-balances-bc
+```
+
+### Install Node version
+
+More information on how to install NVM: https://github.com/nvm-sh/nvm
+
+```bash
+nvm install
+nvm use
+```
+
+### Install Dependencies
+
+```bash
+npm install
+```
+
+#### Build
+
+```bash
+npm run build
+``` 
+
+## 4. Running Locally (dependencies inside of docker)
+
+In this method, we will run all of the core dependencies inside of docker containers, while running the `builtin-ledger-grpc-svc` server on your local machine.
+
+### Startup supporting services
+
+### 4.1 Run all back-end dependencies as part of the Docker Compose
+
+Use [platform-shared-tools docker-compose files](https://github.com/mojaloop/platform-shared-tools/tree/main/packages/deployment/): 
+Follow instructions in the `README.md` files to run the supporting services. Make sure you have the following services up and running:
+
+- infra services : [docker-compose-infra](https://github.com/mojaloop/platform-shared-tools/tree/main/packages/deployment/docker-compose-infra)
+	- mongo
+	- kafka
+	- zoo
+- cross-cutting services : [docker-compose-cross-cutting](https://github.com/mojaloop/platform-shared-tools/tree/main/packages/deployment/docker-compose-cross-cutting)
+	- authentication-svc
+	- authorization-svc
+	- identity-svc
+	- platform-configuration-svc
+
+This will do the following:
+* `docker pull` down any dependencies defined in each `docker-compose.yml` file, and the services.
+* run all of the containers together
+* ensure that all dependencies have started for each services.
+
+**Important:** please make sure you follow the steps available through the **admin-ui** demonstrated in the sections [here](https://github.com/mojaloop/platform-shared-tools/tree/main/packages/deployment/docker-compose-apps#participants) (you can ignore the Account-Lookup part). There is required data to already exist such as **participants** endpoints exists and its accounts have funds available, otherwise the integrations will fail.
```

Comment:
```suggestion
**Important:** please make sure you follow the steps available through the **admin-ui** demonstrated in the sections [here](https://github.com/mojaloop/platform-shared-tools/tree/main/packages/deployment/docker-compose-apps#participants) (you can ignore the Account-Lookup part). Configuration such as **participants** endpoints and its accounts with funds available are pre-configured as part of setup, otherwise the integrations will fail.
```